### PR TITLE
Added a README file

### PR DESCRIPTION
**Pull request by Mailaender**
A followup of https://github.com/OpenRA/ra2/pull/28 that got lost several times during rebasings.

### CODE REVIEW

**Comment by Mailaender** [comment]
Rebased. Now with a Travis build badge as well. https://github.com/OpenRA/ra2/pull/45

**Comment by Phrohdoh** [comment]
I reiterate that "the latest development version" is completely invalid and is not good enough here.
Please use the mechanisms we have in place.
:-1: for that reason.

**Comment by Mailaender** [comment]
This is just about documenting what we have. Feel free to send a pull request to improve the situation.

**Comment by Phrohdoh** [comment]
No, I'm not okay with someone merging a half-complete (and incorrect) PR because you don't want to put proper work into it and are, for some reason, against using the mechanism we spent a lot of back and forth on to get right for this situation.

**Comment by abcdefg30** [comment]
Could you ammend the latest version of https://github.com/OpenRA/ra2/wiki , please?

**Comment by Mailaender** [comment]
Updated.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Red Alert 2 mod for OpenRA
+
+[![Build Status](https://travis-ci.org/OpenRA/ra2.svg?branch=master)](https://travis-ci.org/OpenRA/ra2)
+
+See https://github.com/OpenRA/ra2/wiki for instructions.
```
Comment (abcdefg30) [inline README.md]: Sounds a bit harsh, IMHO, what about
`See [our wiki pages](https://github.com/OpenRA/ra2/wiki) for instructions on how to install and use this.`?

Comment (Mailaender) [inline README.md]: Okay.

**Comment by abcdefg30** [comment]
Looks ok to me now. :+1:

**Pull request merged**